src/toxext.h

```diff
@@ -97,24 +110,31 @@ struct ToxExtPacketList *toxext_packet_list_create(struct ToxExt *toxext,
  * Appends an extension message onto an existing packet. This is the endpoint
  * extensions should use when trying to send their own data.
  */
+ // "existing packet" -> "existing packet list"?
+ // "This is the endpoint extensions" -> "This is the endpoint the extensions"
+ // not sure about "endpoint" in general vs API or function :shrug:
 int toxext_packet_append(struct ToxExtPacketList *packet /*in/out*/,
 			 struct ToxExtExtension *extension, void const *data,
 			 size_t size);
 
 /**
  * Sends and frees an existing packet
  */
+ // "packet" -> "packet list"? was called a "packet list" in toxext_packet_list_create, and that seems more accurate
```

Comment:
Yeah I think this is a good change. I think I renamed Packet -> PacketList fairly late in the process of my initial implementation so some of the naming may be left over

Reply:
Ah I see :+1: